QRCode/Source/QRCode.swift

```diff
@@ -179,7 +181,7 @@ open class QRCode: NSObject, AVCaptureMetadataOutputObjectsDelegate {
             return
         }
         
-        if !session.canAddInput(videoInput) {
+        if !session.canAddInput(videoInput!) {
```

Comment:
Hey, I believe this will crash the app if the user denies camera permission, making `videoInput` `nil`.